lib/screens/explore_screens/vote_screens/delegate/delegators_tab/interactor/mapper/delegator_load_data_mapper.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:seeds/datasource/remote/model/member_model.dart';
+import 'package:seeds/domain-shared/page_state.dart';
+import 'package:seeds/domain-shared/result_to_state_mapper.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegators_tab/interactor/viewmodels/delegator_state.dart';
+
+class DelegatorLoadDataStateMapper extends StateMapper {
+  DelegatorState mapResultToState(DelegatorState currentState, List<Result>? results) {
+    if (results == null) {
+      return currentState.copyWith(pageState: PageState.failure);
+    } else if (results.isEmpty) {
```

Comment:
these 2 else if branches can be combined into 1

hasDelegators: !result.isEmpty